sale_timesheet_line_exclude/readme/USAGE.rst

```diff
@@ -1,11 +1,9 @@
-To exclude a specific timesheet entry from Sale Order:
+To exclude a specific timesheet entry from invoicing:
 
-# Go to *Timesheets > Timesheet > All Timesheets*
-# Check *Exclude From Billing* for a specific timesheet entry
+  # Go to *Timesheets > Timesheet > All Timesheets*
+  # On the Timesheet list view, check *Non-billable* for specific timesheet entries
 
 or:
 
-# Go to *Project > All Tasks*
-# Open specific task for editing
-# Switch to *Timesheets* tab
-# Check *Exclude From Billing* for a specific timesheet entry
+  # Go to *Project > All Tasks*, ans open a specific Task form
```

Comment:
```suggestion
  # Go to *Project > All Tasks*, and open a specific Task form
```

Reply:
@dreispt it should be `#.` and without spaces otherwise it indents it even more

Reply:
You're right. Actually, a regular `- ` should do: https://docutils.sourceforge.io/docs/user/rst/quickref.html#bullet-lists

Reply:
Let's fix that and merge I guess

Reply:
The idea of `#.` is to have a numbered bullet list, so better to stick to this convention than to put `-`